flow/ondemand/client/client.go

```diff
@@ -378,6 +378,21 @@ func (o *OnDemandProbeClient) Stop() {
 	o.checkForRegistration.Stop()
 }
 
+// Invoke capture upon start of analyzer on selected from config SubGraph
+func (o *OnDemandProbeClient) InvokeCaptureFromConfig() () {
+	capture_gremlin := config.GetConfig().GetString("analyzer.startup_capture")
```

Comment:
You are already within the context of a function handling capture so no need to prefix variables with `capture_` form example `gremlin` or `gremlin_filter` may be better

Reply:
ok :)